pkg/volume/util.go

```diff
@@ -439,3 +439,30 @@ func GetPVCMatchExpression(pvc *v1.PersistentVolumeClaim, key string, operator u
 	}
 	return make(sets.String), fmt.Errorf("Persistent volume claim (%v): %v operator for %v key not found in the matchExpressions part", pvc.Name, key, operator)
 }
+
+// Regions2ZonesParams are parameters for the func Regions2Zones
+type Regions2ZonesParams struct {
```

Comment:
I think you don't need an extra structure for 3 parameters. Or do I miss something?